.github/CODEOWNERS

```diff
@@ -1,6 +1,46 @@
 
-# docs                                @eclipse-score/automotive-score-committers
+# /docs                              @eclipse-score/process-community
+# /docs/concepts                     @eclipse-score/automotive-score-committers
+# /docs/glossary                     @eclipse-score/automotive-score-committers
+# /docs/features                     @eclipse-score/technical-lead
+# /docs/features/*/architecture      @eclipse-score/automotive-score-committers
+# /docs/features/*/requirements      @eclipse-score/technical-lead
+# /docs/features/*/safety_analysis   @eclipse-score/safety-manager
+# /docs/features/*/safety_planning   @eclipse-score/safety-manager @eclipse-score/technical-lead
+# /docs/features/*/verification      @eclipse-score/quality-manager @eclipse-score/safety-manager
+# /docs/manual                       @eclipse-score/automotive-score-committers
+# /docs/overview                     @eclipse-score/project-lead
+# /docs/pmp                          @eclipse-score/technical-lead @eclipse-score/process-community
+# /docs/pmp/quality_management       @eclipse-score/quality-manager
+# /docs/pmp/safety_management        @eclipse-score/safety-manager
+# /docs/pmp/security_management      @eclipse-score/security-manager
+# /docs/requirements/stakeholder     @eclipse-score/technical-lead
+# /docs/requirements/tool            @eclipse-score/infrastructure-tooling-community @eclipse-score/process-community
+# /docs/release                      @eclipse-score/quality-manager @eclipse-score/technical-lead
+# /docs/safety                       @eclipse-score/safety-manager @eclipse-score/technical-lead
+# /docs/security                     @eclipse-score/security-manager @eclipse-score/technical-lead
+# /docs/tutorials                    @eclipse-score/automotive-score-committers
+
+# /examples                          @eclipse-score/automotive-score-committers
+# /integration_tests                 @eclipse-score/automotive-score-committers
+# /tools                             @eclipse-score/infrastructure-tooling-community
+
+# in separate process_description repository:
+#
+# /process                  @eclipse-score/process-community
+
+# in separate <module_name> repositories:
+#
+# /docs                     @eclipse-score/process-community
+# /docs/manual              @eclipse-score/<module_name>/safety-manager
+# /docs/release             @eclipse-score/<module_name>/quality-manager @eclipse-score/<module_name>/module-lead
+# /docs/safety_plan         @eclipse-score/<module_name>/safety-manager @eclipse-score/<module_name>/module-lead
+# /docs/safety_analysis     @eclipse-score/<module_name>/safety-manager
+# /docs/verification        @eclipse-score/<module_name>/quality-manager @eclipse-score/<module_name>/safety-manager
+# /components               @eclipse-score/<module_name>/technical-lead
+# /components/*/            @eclipse-score/<module_name>/automotive-score-committers
```

Comment:
@nradakovic @ltekieli when you set up the template repository for modules, could you already add this part there in the CODEOWNERS?

We would probably need a readme there that then tells what steps need to be done manually (e.g. adapting the group names,...)